tests/testthat/test-epidatacall.R

```diff
@@ -142,14 +144,14 @@ test_that("fetch_tbl warns on non-success", {
     .package = "jsonlite"
   )
 
-  expect_warning(epidata_call %>% fetch_tbl(),
+  expect_warning(epidata_call %>% fetch(),
     regexp = paste0("epidata warning: `", artificial_warning, "`"),
     fixed = TRUE
   )
 })
 
-test_that("classic only fetch", {
-  # delphi is an example endpoint that only suports the classic call
+test_that("fetch classic works", {
+  # only_supports_classic is TRUE
   epidata_call <- pub_delphi(
```

Comment:
I don't /think/ refactor changed this, but this outputs a very weird format.  Still trying to be shoehorned into a wide data.frame rather than kept as a nested list structure.